resources/views/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {PageHeader, Card, Row, Col, Divider} from 'antd'
+import {Link} from 'react-router-dom'
+
+const WelcomePage = () => (
+  <>
+    <div style={{}}>
+      <PageHeader
+        className='site-page-header'
+        title='Welcome to Cycl.io' />
+    </div>
+    <Divider>Overview</Divider>
+
+    <Row justify='space-around'>
+      <Col span={8}>
+        <Card hoverable title='Inventory Overview' bordered extra={<Link to='/Production/Inventory'>Open</Link>} style={{width: 400}}>
+          <p>Preview of inventory</p>
+        </Card>
+      </Col>
+      <Col span={8}>
+        <Card hoverable title='Placeholder Overview' bordered extra={<Link to='/Production/Inventory'>Open</Link>} style={{width: 400}}>
```

Comment:
decide if you want to do inline styling to the entire file refactor to a css file later
or you can use `styled-components`